test/Makefile

```diff
@@ -9,7 +9,7 @@ DOCKERCOMPOSEAPPETHTXMANAGER := supernets2-eth-tx-manager
 DOCKERCOMPOSESTATEDB := supernets2-state-db
 DOCKERCOMPOSEPOOLDB := supernets2-pool-db
 DOCKERCOMPOSEEVENTDB := supernets2-event-db
-DOCKERCOMPOSEDACDB := supernets2-data-availability-db
+DOCKERCOMPOSEDACDB := supernets2-data-node-db
```

Comment:
any reason for this rename? per my previous comment, we should keep the changes minimum

Reply:
upstream uses this name

Reply:
what I am suggesting is that we don't need to create this alias at all so the code can be simpler